mautic/mautic
# Report line and pie charts styled to look the same as the rest

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Kinda
| BC breaks?    | No
| Deprecations? | No
| Fixed issues  |  N/A

## Description
This PR changes the styling of the Report charts to look the same as the rest of charts in Mautic 1.4

## Steps to reproduce the bug (if applicable)
Open a report where you have some line and pie charts. The lines should be thick, points quite big and the line chart fill color dark.

## Steps to test this PR
Apply this PR and the lines should be thinner, the point smaller and the line chart fill color lighter.
